src/components/computing-status/computing-type.ts

```diff
@@ -17,4 +17,8 @@ export enum ComputingType {
     STATE_ESTIMATION = 'STATE_ESTIMATION',
 }
 
+export const isValidComputingType = (value: string): boolean => {
```

Comment:
Maybe rename this `ComputationType` instead of `ComputingType` like in the backend...

Reply:
Done.

Reply:
This involve to much changes, Maybe to do in another PR